CHANGELOG.md

```diff
@@ -34,12 +34,14 @@ Dropping a requirement of a major version of a dependency is a new contract.
 - Update jira configuration if ApacheLoadBalancer is chosen. Fix [JPERF-506]
 - Fix `Ec2VirtualUsersFormula` network. Fix [JPERF-406].
 - Fix `StandaloneFormula` network. Fix [JPERF-516].
+- Parallelize AwsCli#ensureAwsCli across SSH hosts. Fix [JPERF-518].
```

Comment:
2.12.0 was release in the meantime

Reply:
Waiting 4 hours for a PR only to see that conflict 😞.

Reply:
https://circleci.com/gh/wyrzyk/aws-infrastructure/54 reports 11 minutes
If it's really 4 hours, then I'd trade 3+ hours dev loop improvement for this increase in test coverage any day.l